client/src/components/ModalSave.jsx

```diff
@@ -4,37 +4,128 @@ import React from 'react';
 
 import styled from 'styled-components';
 
-const ShareModalBox = styled.div`
+const SaveModalBox = styled.div`
   width: 500px;
   text-align: center;
   background-color: #ffffff;
-  padding: 10px 0px;
   border-radius: 10px;
 `;
 
+const Head = styled.div`
+  padding: 16px 24px;
+  border-bottom: 1px solid #ebebeb;
+`;
+
+const CloseButton = styled.div`
+  float: left;
+  height: 24px;
+  width: 24px;
+  background-image: url("./Images/badge-images/close.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-bottom: 24px;
+  cursor: pointer;
+`;
+
 const Heading = styled.div`
   font-weight: bold;
   display: inline-block;
   margin-top: 5px;
+  font-size: 16px;
 `;
 
-const CrossButton = styled.div`
-  float: left;
-  margin-left: 20px;
-  text-align: left;
-  font-size: 1.4em;
-  font-weight: bold;
+const Body = styled.div`
+  padding: 24px;
+`;
+
+const InputBoxes = styled.div`
+  border-radius: 8px;
+  border: 1px solid #b0b0b0;
+`;
+
+const CountryBox = styled.div`
+  display: block;
+  height: 56px;
+`;
+
+const PhoneBox = styled.div`
+  display: block;
+  height: 56px;
+`;
+
+const InputBoxesSubText = styled.div`
+  font-size: 12px;
+  color: #717171;
+  margin-bottom: 16px;
+`;
+
+const ContinueButton = styled.div`
+  height: 48px;
+  width: 100%;
+  line-height: 48px;
+  margin-bottom: 16px;
+  color: ${props => props.textColor};
+  background-color: ${props => props.backgroundColor};
+  background-image: url("./Images/badge-images/social/${props => props.backgroundImage}");
+  background-size: 25px 25px;
+  background-position: 12px 10px; 
+  background-repeat: no-repeat;
+
+  border: 1px solid ${props => props.borderColor};
+  border-radius: 8px;
+
+  &:hover {
+    border: 1px solid ${props => props.borderHoverColor};
+  }
+`;
+
+const Footer = styled.div`
+
+  span {
+    font-size: 14px;
+    color: #484848;
+  }
+
+  button {
+    border: 0;
+    cursor: pointer;
+    font-size: 14px;
+    color: #222222;
+    text-decoration: underline;
+  }
 `;
```

Comment:
I would recommend moving this css to another file.

Reply:
Will learn and do that soon